frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellDrawer.spec.tsx

```diff
@@ -40,7 +40,7 @@ describe('CloudShellDrawerComponent', () => {
       .find(Drawer)
       .shallow()
       .find(Button);
-    expect(buttons.at(1).props()['aria-label']).toEqual('Close Terminal');
+    expect(buttons.at(1).props()['aria-label']).toEqual('Close terminal');
```

Comment:
This is where having a `data-test-id` would be beneficial.

Reply:
using data-test-id